src/main/java/com/programmerdan/minecraft/cropcontrol/config/RootConfig.java

```diff
@@ -185,6 +197,15 @@ public String predictDrops(BreakType breakType, UUID placer, UUID breaker, boole
 				localMax += modifier.stackExpand;
 			}
 			
+			//per world support
+			if (dropMod.worlds.containsKey(world.getName())) { // world
```

Comment:
I wondered how you would approach this, I like it. Simple, direct. Easy to disable specific worlds, or have "high growth" worlds. Bravo!